screens/TabOneScreen.tsx

```diff
@@ -49,8 +50,19 @@ export default function TabOneScreen ({ navigation }) {
     );
   };
 
+
   function onEndReached (){
-    alert('End reached!');
+    console.log(page);
+    if (page<5){
+      setPage(page + 1)
+      console.log('https://swapi.dev/api/vehicles/?page=' + String(page))
```

Comment:
w JS możesz używać takiej składni:
```suggestion
      console.log(`https://swapi.dev/api/vehicles/?page=${page}`)
```